### PR TITLE
Creating QR code reader document

### PR DESCRIPTION
**Pull request by jpwalden**
Signed-off-by: jari-pekka walden <jari-pekka.walden@jolla.com>

### CODE REVIEW

**Review by jovirkku — APPROVED** [review]
LGTM.  Nothing to change.

**Comment by vigejolla** [comment]
Nitpick: Please change the commit message to imperative form: "Create QR code reader document"

File: Support/Help_Articles/QR_code_Reader/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: QR Code Reader
+permalink: Support/Help_Articles/QR_Code_Reader/
```
Comment (vigejolla) [inline Support/Help_Articles/QR_code_Reader/README.md:3]: The title and permalink have capital C, while the directory name has lower case c. I suggest you rename the directory.

**Review by vigejolla — CHANGES_REQUESTED** [review]

File: Support/Help_Articles/QR_code_Reader/README.md (hunk 1 above)
Comment (jpwalden) [reply Support/Help_Articles/QR_code_Reader/README.md]: Ok, renamed.

**Pull request merged**